### DISCUSSION TITLE
Getting error for closed braces which says unclosed

### DISCUSSION THREAD

**Discussion by xtheeq** [wezterm/wezterm#5904] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

my code:
```
local wezterm = require("wezterm")
local mux = wezterm.mux

local config = {}

config.default_prog = { "C:\\Users\\athee\\scoop\\apps\\nu\\current\\nu.exe" }
config.enable_tab_bar = false 
config.window_decorations = "RESIZE"
config.window_padding = {left = 3, top = 8, bottom = 0, right = 0}
config.color_scheme = "Catppuccin Mocha"
config.font = wezterm.font("JetBrains Mono")
config.font_size = 12

config.keys = {
  {
    key = "f",
    mods = "CTRL|CMD",
    action = wezterm.action.ToggleFullScreen,
  },
},

return config

```

error:
`
syntax error: [string "C:\Users\athee\.config\wezterm\wezterm.lua"]:18: '}'
expected (to close '{' at line 15) near 'action'
`

I'm using nushell does that make any prob?

**Comment by bew** [comment, marked as answer]
Try to remove the comma after the very last closing brace, the one for `config.keys`.

(nushell is not relevant here)

**Reply by xtheeq** [reply]
yeah that made the prob 
i asked abt nushell cuz it made a issue with wezterm
